src/Components/CreateImageWizard/validators.ts

```diff
@@ -48,8 +48,8 @@ export const isMountpointMinSizeValid = (minSize: string) => {
 export const isBlueprintNameValid = (blueprintName: string) =>
   blueprintName !== undefined &&
   blueprintName.length >= 2 &&
-  blueprintName.length <= 100 &&
-  /\w+/.test(blueprintName);
+  blueprintName.length <= 300 &&
+  /[a-zA-Z0-9]{2,}/.test(blueprintName);
```

Comment:
This changes the non-asci characters from being valid to not being valid. Was that an intention?